docs/ballerina-by-example/examples/http-failover/http-failover.bal

```diff
@@ -0,0 +1,94 @@
+import ballerina.net.http;
+import ballerina.net.http.resiliency;
+import ballerina.mime;
+import ballerina.runtime;
+
+service<http> failoverService {
+
+    http:Retry retryConfiguration = {count:0};
+
+    // Set of HttpClient connectors to be failover.
+    http:HttpClient endPoint1 = create http:HttpClient("http://localhost:909990/mock",
+                                                           {keepAlive:false});
+    http:HttpClient endPoint2 = create http:HttpClient("http://localhost:9090/echo",
+                                                       {endpointTimeout:5000,
+                                                           keepAlive:false,
+                                                           retryConfig:retryConfiguration});
+    http:HttpClient endPoint3 = create http:HttpClient("http://localhost:9090/mock",
+                                                           {keepAlive:false});
+
+    http:HttpClient[] failoverGroup = [endPoint1, endPoint2, endPoint3];
+
+    // Set of Http Status codes needs to be failover.
+    int[] errorCodes = [400, 404, 500];
+
+    resiliency:FailoverConfig errorCode = {failoverCodes:errorCodes};
+
+    @Description {value:"Requests which contain any HTTP method will be directed to passthrough resource."}
+    @http:resourceConfig {
+        path:"/"
+    }
+
+    resource failoverPostResource (http:Connection conn, http:InRequest req) {
+        // Failover Connector takes the input as an array of HttpClient connectors and FailoverConfig struct.
+        // The FailoverConfig struct should contain int array of http status codes and interval of failover in  milliseconds.
+        //      struct FailoverConfig {
+        //             int[] failoverCodes;
+        //             int interval;
+        //      }
+        endpoint<http:HttpClient> ep {
+            create resiliency:Failover(failoverGroup, errorCode);
+        }
+
+        http:InResponse clientResponse = {};
+        http:HttpConnectorError err;
+
+        mime:MediaType contentType = mime:getMediaType("application/xml");
+        mime:Entity requestEntity = {};
+        requestEntity.setXml( xml `<name>Ballerina</name>`);
+        requestEntity.contentType = contentType;
+
+        http:OutRequest outReq = {};
+        outReq.setEntity(requestEntity);
+
+        clientResponse, err = ep.post("/", outReq);
+
+        http:OutResponse res = {};
+        if (err != null) {
+            res.statusCode = err.statusCode;
+            res.setStringPayload(err.message);
+            _ = conn.respond(res);
```

Comment:
Shall we explicitly assign this error as well? Since we agreed not to ignore errors in Ballerina by Example examples

Reply:
Ack